sub-pages/_single.html

```diff
@@ -70,17 +70,17 @@
     {% endif %}
 
     {% if activities_feed and display_activity_flag == true %}
-    <section class="block
-                    block__flush-sides
-                    block__bg
-                    u-mb0
-                    u-mt0">
-        {% include 'templates/activities-feed.html' %}
-        <a class="jump-link jump-link__right"
-           href="/activity-log/{{ ('?filter_tags=' ~ sub_page.tags | join('&') ) if sub_page.tags }}">
-            View all of our activities
-        </a>
-    </section>
+        <section class="block
+                        block__flush-sides
+                        block__bg
+                        u-mb0
+                        u-mt0">
```

Comment:
Can `block__flush-top` and `block__flush-bottom` be used here? (Double-check, because they aren't completely identical behavior.